ext/dom/php_dom.stub.php

```diff
@@ -889,7 +889,7 @@ public function prepend(...$nodes): void {}
         public function replaceChildren(...$nodes): void {}
     }
 
-    /** @alias DOM\DOMException */
+    /** @alias Dom\DOMException */
```

Comment:
Is it intentional that the class name follows the discouraged pattern of uppercased abbreviations? Is it for internal consistency with the existing pattern? Personally, I'd vote for going full steam PascalCase, This applies for other abbreviations like `HTTP`.

Reply:
The official name is DOMException: https://webidl.spec.whatwg.org/#idl-DOMException and so this was left alone.

Reply:
Ah ok, I see. Then I guess the same applies to all the other names which were left intact.